iotdb-1.1/src/main/java/cn/edu/tsinghua/iot/benchmark/iotdb110/IoTDB.java

```diff
@@ -305,18 +305,14 @@ private void registerStorageGroups(Session metaSession, List<TimeseriesSchema> s
   private void activateTemplate(Session metaSession, List<TimeseriesSchema> schemaList) {
     List<String> someDevicePaths = new ArrayList<>();
     AtomicLong activatedDeviceCount = new AtomicLong();
-    schemaList.stream()
-        .map(schema -> ROOT_SERIES_NAME + "." + schema.getDeviceSchema().getDevicePath())
-        .forEach(
-            path -> {
-              someDevicePaths.add(path);
-              if (someDevicePaths.size() >= ACTIVATE_TEMPLATE_THRESHOLD) {
-                activateTemplateForSomeDevices(
-                    metaSession, someDevicePaths, activatedDeviceCount.get());
-                activatedDeviceCount.addAndGet(someDevicePaths.size());
-                someDevicePaths.clear();
-              }
-            });
+    for (TimeseriesSchema timeseriesSchema : schemaList) {
+      someDevicePaths.add(timeseriesSchema.getDeviceId());
+      if (someDevicePaths.size() >= ACTIVATE_TEMPLATE_THRESHOLD) {
+        activateTemplateForSomeDevices(metaSession, someDevicePaths, activatedDeviceCount.get());
+        activatedDeviceCount.addAndGet(someDevicePaths.size());
+        someDevicePaths.clear();
+      }
+    }
```

Comment:
Please check `map` operations, I think this change may cause some problem

Reply:
I have checked the device names created by template
![image](https://github.com/thulab/iot-benchmark/assets/38746920/84737b53-ba7e-4392-a698-2f2db79cc415)